src/components/Dialogs/CreateTdDialog.jsx

```diff
@@ -15,13 +15,18 @@ import ReactDOM from "react-dom";
 import { ChevronDown } from "react-feather";
 import ediTDorContext from "../../context/ediTDorContext";
 import { DialogTemplate } from "./DialogTemplate";
+import { parseCsv, mapCsvToProperties } from "../../utils/parser";
 
 export const CreateTdDialog = forwardRef((props, ref) => {
   const context = useContext(ediTDorContext);
   const [display, setDisplay] = React.useState(() => {
     return false;
   });
   const [type, setType] = React.useState("TD"); // either TD or TM
+  const [properties, setProperties] = React.useState({});
+  const [fileName, setFileName] = React.useState("");
+  const [protocol, setProtocol] = React.useState("Modbus TCP");
```

Comment:
The protocol is not really used currently only displayed, right? Which makes sense as we only support Modbus atm. but just wanted to ask anyway.